frontend/packages/dev-console/src/components/topology/actions/workloadActions.ts

```diff
@@ -29,9 +36,44 @@ export const workloadActions = (workload: TopologyDataObject): KebabOption[] =>
     case 'DaemonSet':
       menuActions = daemonSetMenuActions;
       break;
+    case EventSourceCronJobModel.kind:
+      menuActions = [
+        ...Kebab.getExtensionsActionsForKind(EventSourceCronJobModel),
+        ...Kebab.factory.common,
+      ];
+      break;
+    case EventSourceContainerModel.kind:
+      menuActions = [
+        ...Kebab.getExtensionsActionsForKind(EventSourceContainerModel),
+        ...Kebab.factory.common,
+      ];
+      break;
+    case EventSourceApiServerModel.kind:
+      menuActions = [
+        ...Kebab.getExtensionsActionsForKind(EventSourceApiServerModel),
+        ...Kebab.factory.common,
+      ];
+      break;
+    case EventSourceCamelModel.kind:
+      menuActions = [
+        ...Kebab.getExtensionsActionsForKind(EventSourceCamelModel),
+        ...Kebab.factory.common,
+      ];
+      break;
+    case EventSourceKafkaModel.kind:
+      menuActions = [
+        ...Kebab.getExtensionsActionsForKind(EventSourceKafkaModel),
+        ...Kebab.factory.common,
+      ];
```

Comment:
I think obtaining actions via the extensions as the default would work instead of creating separate cases for these. Since all actions from now on should be contributed through extensions the mechanism is generic.

Also, if using this action provider to supply actions for more than just `workloads` it's probably worth renaming it and related variables now too.